### ISSUE TITLE
`export_gtf` returns only exons

### ISSUE THREAD

**Issue by sbresnahan** [Genentech/Isosceles#8]
As the title notes, a GTF returned by `export_gtf` is incomplete and should contain entries for transcripts.

**Comment by timbitz** [comment]
Hi-- This seems reasonable I guess.  Though I must admit, I was always pretty amused by those, as they are redundant info in the GTF format, since exon entries also have both transcript_id and gene_id entries.  In fact I actually don't think transcript entries are required in GTF2.2 format (http://mblab.wustl.edu/GTF22.html).  Is some downstream program throwing an error that they are missing?

**Comment by sbresnahan** [comment]
Yes, downstream programs may require transcript entries - e.g., [SQANTI3](https://github.com/ConesaLab/SQANTI3).
